.github/workflows/go.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           fetch-depth: 2
       - uses: actions/setup-go@v5
         with:
-          go-version: '1.17'
```

Comment:
This should probably match the version given in the README and in [go.mod](go.mod)

Reply:
tru